frontend/packages/component-library/src/common/index.ts

```diff
@@ -1,3 +1,3 @@
-export * as types from './types';
+export * from './types';
```

Comment:
Why change this?

Reply:
The usage of these types in `apps` were something like:

```
import { Definition } from '@cdo/apps/componentLibrary/common/types

interface A extends Definition {}
```

But this was exported such that you have to do

```
import { types } from '@cdo/apps/componentLibrary/common/types

interface A extends types.Definition {}
```

So this change makes it compatible with the prior approach (not needing a separate constant but spreading directly)

Reply:
Synced up offline and discussed creating entrypoints for sub-directories in `common` rather than a single one for `common`. Updated in this [commit](https://github.com/code-dot-org/code-dot-org/pull/63481/commits/4a4207fc6558d6ce6e640aac0784de2c924412d7)